subsys/usb/usb_device.c

```diff
@@ -233,12 +233,22 @@ static bool usb_handle_request(struct usb_setup_packet *setup,
  *
  * @return N/A
  */
-static void usb_data_to_host(void)
+static void usb_data_to_host(u16_t len)
 {
 	u32_t chunk = usb_dev.data_buf_residue;
 
+#if CONFIG_USB_NRF52840
```

Comment:
You need to be more specific, reel_board does not need this hack

Reply:
That's odd, there should be nothing board specific in this case. I will try to get hold of a reel_board and test it.

Reply:
I also really do not like placing this code to common `usb_device.c`, any change this can be moved to device controller?

Reply:
![reel_board](https://user-images.githubusercontent.com/3072104/56715071-47665480-673f-11e9-8fb0-2435122b0d53.png)

Reply:
For reel_board looks OK

Reply:
The issue exists on Windows only (7 & 10). I've tried reel_board and was able to reproduce it:

![reel_board_zlp](https://user-images.githubusercontent.com/42532799/56718129-7a5c1880-673e-11e9-98d9-da4db9c3a3bc.png)